hvplot/converter.py

```diff
@@ -599,9 +599,9 @@ def _process_symmetric(self, symmetric, clim, check_symmetric_max):
 
         if is_xarray(self.data):
             # chunks mean it's lazily loaded; nanquantile will eagerly load
-            if self.data.chunks:
-                return False
             data = self.data[self.z]
+            if not getattr(data, '_in_memory', True) or data.chunks:
+                return False
```

Comment:
This seems reasonable to me insofar as it treats non-in-memory data the same as chunked data